tests/one_wire_test.toit

```diff
@@ -7,6 +7,7 @@ import expect show *
 import monitor
 
 main:
+  test-search-2
   test-search
```

Comment:
```suggestion
  test-search
  test-search-2
```

Reply:
I was wondering if you would make the comment :)